src/Underscores.jl

```diff
@@ -102,9 +102,7 @@ const _pipeline_ops = [:|>, :<|, :∘, :.|>, :.<|]
 
 function lower_underscores(ex)
     if ex isa Expr
-        if isquoted(ex)
```

Comment:
I don't think this should be removed — it currently ignores `Expr(:meta, ...)` which it should continue doing (and `Expr(:inert)` I guess... though `:meta` and particularly `:inert` are quite obscure!)